packages/react/src/components/Table/TableHeader.tsx

```diff
@@ -69,6 +69,12 @@ const TableHeader = forwardRef<HTMLTableHeaderCellElement, TableHeaderProps>(
             onClick={onSort}
             className="TableHeader__sort-button"
             type="button"
+            style={{
```

Comment:
Generally speaking, we want to avoid inline styles.

For the grid layout, it's a little unavoidable due to the dynamic nature of the columns, and those styles are already set here: https://github.com/dequelabs/cauldron/blob/abc96db6f05e266457ccca23a2b0f83fd879f1ee/packages/react/src/components/Table/TableHeader.tsx#L64

Rather than inlining new styles, we should take advantage of CSS inheritance to achieve the same results. We should be able to do this by removing `display: flex` and setting `text-align: inherit` here: https://github.com/dequelabs/cauldron/blob/abc96db6f05e266457ccca23a2b0f83fd879f1ee/packages/styles/table.css#L78

Reply:
Sadly that does not fix it 
<img width="798" height="612" alt="image" src="https://github.com/user-attachments/assets/22caccd1-68a3-46e1-b986-d80d2a2baa68" />

Reply:
You are correct. Please excuse that oversight, however, this still does not work when align end is set on the column

<img width="1248" height="584" alt="image" src="https://github.com/user-attachments/assets/512c60bb-b651-4e3d-a92c-6d149fdf83a6" />

Reply:
This was the change I made in the above linked `table.css` file. 

```diff
  font-size: var(--text-size-small);
  font-weight: var(--font-weight-medium);
  color: var(--table-header-text-color);
- display: flex;
+ text-align: inherit;
  align-items: center;
  justify-content: flex-start;
  padding: var(--table-space) var(--space-large);
}
```

I made a slight modification to the docs page to display a sortable column within the grid and this is what I'm seeing:

<img width="529" height="391" alt="cauldron docs page, table" src="https://github.com/user-attachments/assets/9dd5f0f1-2f11-4139-8f85-b11e53e21b66" />

Are you doing something different?

Reply:
I had a different property still applied locally. Thank you for patience!